terms-and-conditions.md

```diff
@@ -12,14 +12,14 @@ layout: page
 <li>By agreeing to have the session included in the programme the session leader is committing to participate in the shepherding process to ensure high session quality</li>
 </ol>
 
-<p>The session leader grants SPA the right to reproduce and distribute materials submitted for the session for the programme (online and printed). The copyright of the materials remains with the original copyright holder. Where copyright is held by a person other than the session leader, it is the session leader’s responsibility to ensure appropriate permission to use is secured.</p>
+<p>The session leader grants the SPA Specialist Group the right to reproduce and distribute materials submitted for the session for the programme (online and printed). The copyright of the materials remains with the original copyright holder. Where copyright is held by a person other than the session leader, it is the session leader’s responsibility to ensure appropriate permission to use is secured.</p>
```

Comment:
I assume that this should refer to the SPA group rather than the conference - can anyone confirm? @eoinwoods @NTCoding

Reply:
Can we replace it with 'the conference organisers'? As it's us - SPA conf organisers - who distribute the online material, and Mandy at BCS who prints the programme.

Reply:
How about "The Conference" ? Refer to the entity than the people behind it.

Reply:
Thanks - I've gone with 'the conference organisers', I think it's more precise and consistent with the rest of the content.